package-lock.json

```diff
@@ -0,0 +1,8185 @@
+{
```

Comment:
Why add the `package_lock.json`?

Reply:
Because it ensures the versions of (nested) dependencies remain consistent across environments which is also usually a recommended practice when it comes to security.